apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/app/GenesisImporterTest.java

```diff
@@ -139,30 +131,52 @@ void setUp() {
     void newGenesisBlock() {
         doReturn("conf/data/genesisParameters.json").when(genesisImporterProducer).genesisParametersLocation();
         doReturn("conf/data/genesisAccounts-testnet.json").when(chain).getGenesisLocation();
-        genesisImporter = new GenesisImporter(blockchainConfig, configDirProvider, blockchainConfigUpdater,
-                extension.getDatabaseManager(), aplAppStatus, genesisImporterProducer);
+        genesisImporter = new GenesisImporter(
+                blockchainConfig,
+                blockchainConfigUpdater,
+                extension.getDatabaseManager(),
+                aplAppStatus,
+                genesisImporterProducer,
+                weld.select(ApplicationJsonFactory.class).get()
+        );
         genesisImporter.loadGenesisDataFromResources(); // emulate @PostConstruct
 
         Block block = genesisImporter.newGenesisBlock();
         assertNotNull(block);
         assertEquals(1739068987193023818L, block.getGeneratorId());
-        assertEquals(4997592126877716673L, block.getId());
+        assertEquals(getOSDependentExpectedBlockId(), block.getId());
```

Comment:
Why is the block id different?

Reply:
While it's working fine in Linux (returns 4997592126877716673L), it returns a different long in Windows. If one tries to run this test on develop branch on Windows now, it fails.

Reply:
fixed